<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>inicio_dialog</class>
 <widget class="QDialog" name="inicio_dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1366</width>
    <height>768</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <property name="windowIcon">
   <iconset resource="pics.qrc">
    <normaloff>:/images/robot.png</normaloff>:/images/robot.png</iconset>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255,255,255);</string>
  </property>
  <widget class="QPushButton" name="pushButton_recommend">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>420</y>
     <width>1041</width>
     <height>281</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="styleSheet">
    <string notr="true"> background-color: #4ae126; /* Green */
  border: none;
  color: white;
  padding: 15px 32px;
  text-align: left;
  text-decoration: none;
  display: inline-block;
  font-size: 20px;
  font-weight: bold;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="iconSize">
    <size>
     <width>120</width>
     <height>120</height>
    </size>
   </property>
   <property name="autoDefault">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>450</y>
     <width>1001</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255,255, 255);
font-size: 25px;
 background-color: transparent;</string>
   </property>
   <property name="text">
    <string>Pensamos que quizá podrías revisar estos temas para mejorar tu aprendizaje...</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_progress">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>120</y>
     <width>521</width>
     <height>291</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="iconSize">
    <size>
     <width>120</width>
     <height>120</height>
    </size>
   </property>
   <property name="shortcut">
    <string/>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_learn">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>120</y>
     <width>511</width>
     <height>291</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">content: url(:/images/objetivo.png);
background-color: #ffeb3b; /*grey*/
 background-position: center;
border: yellow;
color: white;
padding: 15px 32px;
text-align: left;
text-decoration: none;
 display: inline-block;
  font-size: 20px;
  font-weight: bold;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="iconSize">
    <size>
     <width>120</width>
     <height>120</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>230</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);
font-size: 25px;
background-color: transparent; /*grey*/</string>
   </property>
   <property name="text">
    <string>Continúa</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>260</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);
font-size: 25px;
content: url(:/images/objetivo.png);
background-color: transparent; /*grey*/</string>
   </property>
   <property name="text">
    <string>aprendiendo</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>890</x>
     <y>220</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);
font-size: 25px;
 background-color: transparent; /*grey*/
</string>
   </property>
   <property name="text">
    <string>¡Mira tu</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>890</x>
     <y>250</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);
font-size: 25px;
 background-color: transparent; /*grey*/
</string>
   </property>
   <property name="text">
    <string>progreso!</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>119</y>
     <width>291</width>
     <height>581</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"> background-color: #f5f5f5; /*grey*/</string>
   </property>
   <widget class="QLabel" name="user_name">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>50</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size: 25px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="chat_box">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>180</y>
      <width>281</width>
      <height>201</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="pics.qrc">:/images/chat_box.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="user_icon">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>91</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size: 25px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="chat_bot">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>340</y>
      <width>121</width>
      <height>111</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="pics.qrc">:/images/chatbot.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_config">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>440</y>
      <width>291</width>
      <height>71</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true"> background-color: transparent;
  border: none;
  color: black;
  padding: 15px 32px;
  text-align: left;
  text-decoration: none;
  display: inline-block;
  font-size: 20px;
</string>
    </property>
    <property name="text">
     <string> Configuración</string>
    </property>
    <property name="icon">
     <iconset resource="pics.qrc">
      <normaloff>:/images/settings.png</normaloff>:/images/settings.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>32</width>
      <height>37</height>
     </size>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_ayuda">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>500</y>
      <width>291</width>
      <height>61</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true"> background-color: transparent;
  border: none;
  color: black;
  padding: 15px 32px;
  text-align: left;
  text-decoration: none;
  display: inline-block;
  font-size: 20px;
</string>
    </property>
    <property name="text">
     <string>  Ayuda</string>
    </property>
    <property name="icon">
     <iconset resource="pics.qrc">
      <normaloff>:/images/ayuda.png</normaloff>:/images/ayuda.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>34</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>230</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>¡Hey!</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>260</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>¡Qué bien que llegaste!</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>200</y>
     <width>131</width>
     <height>121</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: transparent;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="pics.qrc">:/images/objetivo.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>1120</x>
     <y>210</y>
     <width>151</width>
     <height>151</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
 background-color: transparent; /*grey*/

</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="pics.qrc">:/images/grafico_de_barras.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>940</x>
     <y>550</y>
     <width>151</width>
     <height>151</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"> background-color:transparent</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="pics.qrc">:/images/abaco.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>0</y>
     <width>391</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Black</family>
     <pointsize>45</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 85, 255);</string>
   </property>
   <property name="text">
    <string>SuuGemu</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>10</y>
     <width>51</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: transparent;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="pics.qrc">:/images/robot.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>40</y>
     <width>941</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
     <strikeout>false</strikeout>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="text">
    <string>Sistema de Videojuego Educativo Para el Área de Estudio de Matemáticas</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="pics.qrc"/>
 </resources>
 <connections/>
</ui>
